### DISCUSSION TITLE
"Not a diagram file" error when opening file from GitHub

### DISCUSSION THREAD

**Discussion by TheNinth7** [jgraph/drawio#4015] [General]
Hi,

I am trying to convert a Visio file to drawio and store it in a GitHub repository. The Visio file opens just fine in https://app.diagrams.net, and I can save it to my GitHub repository. However when I then try to open it up again I get the error "not a diagram file":

![Screenshot - 01_12_2023 , 21_20_54](https://github.com/jgraph/drawio/assets/118298758/54c63029-5776-4bed-aaeb-9f173b574f6c)

When I click on any of the two .drawio files, I only get this error:

![Screenshot - 01_12_2023 , 21_21_18](https://github.com/jgraph/drawio/assets/118298758/0273139a-2c2e-4b55-a1a9-fd8e804acfcd)

Any suggestion what I could be doing wrong?

**Comment by davidjgraph** [comment]
What is the file suffix for the file being opened?

**Reply by TheNinth7** [reply]
The suffix is .drawio

**Reply by TheNinth7** [reply]
When I download the file from GitHub and open it from my local hard drive, then drawio opens it just fine. Only opening via GitHub leads to the error shown above.
